Makefile.work

```diff
@@ -143,6 +152,36 @@ ifneq ($(SONIC_DPKG_CACHE_SOURCE),)
 	DOCKER_RUN += -v "$(SONIC_DPKG_CACHE_SOURCE):/dpkg_cache:rw"
 endif
 
+# If SONIC_CONFIG_NATIVE_DOCKERD_SHARED is enabled, force
+# SONIC_CONFIG_USE_NATIVE_DOCKERD_FOR_BUILD enabled as well.
+ifeq ($(strip $(SONIC_CONFIG_NATIVE_DOCKERD_SHARED)),y)
+override SONIC_CONFIG_USE_NATIVE_DOCKERD_FOR_BUILD := y
+endif
+
+# User name and tag for "docker-*" images created by native dockerd mode.
+ifeq ($(strip $(SONIC_CONFIG_NATIVE_DOCKERD_SHARED)),y)
+DOCKER_USERNAME = -$(USER)
```

Comment:
incase the user is mixed case, this will be a problem. can you convert user to all lower case?

Reply:
Fixed it.